### DISCUSSION TITLE
How come no one from the GNUstep project was even consulted?

### DISCUSSION THREAD

**Discussion by gcasa** [helloSystem/hello#289] [General]
I didn't even know that GNUstep was considered for this, never mind rejected.  Given that I am the lead developer and that is WELL KNOWN it's surprising to say the least.   I would ask, at least as a matter of mutual respect, that you remove the negative comment about us from your wiki as we were never even asked.

GREAT effort has been put into GNUstep "moving forward" this is evidenced by the push to be compatible with 10.15 or greater over the last 3 years and GNUstep's work with several companies to help port their applications to other platforms. 
 
We could have at least had a discussion about this, but not even a post was made to the mailing lists nor ANY of the developers on it consulted.   So, your OS looks like macOS, but has none of the elegance underneath. 

Yours Respectfully, GC

**Comment by probonopd** [comment]
Thanks for posting @gcasa and sorry if my wording in the wiki was misleading; I changed it immediately. Let's continue in 
https://github.com/helloSystem/hello/discussions/291 where I answered more throughly.
